lang/python312/options.mk

```diff
@@ -18,8 +18,9 @@ PLIST.dtrace=		yes
 PLIST.tkinter=	yes
 .include "../../lang/tcl/buildlink3.mk"
 .include "../../x11/tk/buildlink3.mk"
+CFLAGS+=	-I${PREFIX}/include/tcl/generic
```

Comment:
`BUILDLINK_PREFIX.tcl`, but also this should maybe be added to `BUILDLINK_INCDIRS.tcl` if it's a common problem.